src/fileActions.jsx

```diff
@@ -625,3 +625,4 @@ export const updateFile = (file, currentPath) => {
                     return file;
             }, exc => console.error(`Adding file ${file} failed: ${exc.toString()}`));
 };
```

Comment:
Unrelated change, please remove.

Reply:
what change? I didn't change anything in this file

Reply:
I see a new line was added. Probbaly your editor did it.

Reply:
Ah! got it thanks

Reply:
I get an eslint error when I remove that last line. So I thought it should be this way